src/main/webapp/resources/js/components/ncbi/details/NcbiExportDetailsView.tsx

```diff
@@ -0,0 +1,167 @@
+import { SwapOutlined, SwapRightOutlined } from "@ant-design/icons";
+import { Avatar, Card, List, Space, Tag, Typography } from "antd";
+import React from "react";
+import { useLoaderData } from "react-router-dom";
+import { BasicList } from "../../lists";
+import type { SequenceFile, SingleEndSequenceFile } from "../../../types/irida";
+import { NcbiSubmission, PairedEndSequenceFile } from "../../../types/irida";
+import {
+  BioSampleFileDetails,
+  formatNcbiSubmissionDetails,
+  formatNcbiBioSampleFiles,
+} from "./utils";
+import { formatInternationalizedDateTime } from "../../../utilities/date-utilities";
+import { getNcbiSubmission } from "../../../apis/export/ncbi";
+import { DataFunctionArgs } from "@remix-run/router/utils";
+import { SPACE_LG } from "../../../styles/spacing";
+import { BasicListItem } from "../../lists/BasicList";
+
+/**
+ * React router data loader (https://beta.reactrouter.com/en/dev/route/loader)
+ * Fetches the submission details and formats them to be used by the component.
+ * @param params
+ */
+export async function loader({
+  params,
+}: DataFunctionArgs): Promise<[BasicListItem[], BioSampleFileDetails[]]> {
+  const { id, projectId } = params;
+  if (projectId && id) {
+    return getNcbiSubmission(parseInt(projectId), parseInt(id)).then(
+      (
+        submission: NcbiSubmission
+      ): [BasicListItem[], BioSampleFileDetails[]] => {
+        const { bioSampleFiles, ...info } = submission;
+        const details = formatNcbiSubmissionDetails(info);
+        const bioSamples = formatNcbiBioSampleFiles(bioSampleFiles);
+        return [details, bioSamples];
+      }
+    );
+  } else {
+    return Promise.reject("No project id or export id");
+  }
+}
+
+/**
+ * Render the details of a NCBI SRA Submission
+ * @constructor
+ */
+function NcbiExportDetailsView(): JSX.Element {
+  const [details, bioSampleFiles] = useLoaderData();
+
+  return (
+    <Space direction="vertical" style={{ width: `100%` }}>
+      <Typography.Title level={4} style={{ color: `var(--grey-7)` }}>
+        {i18n("NcbiExportDetailsView.title")}
+      </Typography.Title>
+      <Card>
+        <BasicList dataSource={details} grid={{ gutter: 16, column: 2 }} />
+      </Card>
+      <Typography.Title
+        level={5}
+        style={{ color: `var(--grey-7)`, marginTop: SPACE_LG }}
+      >
+        {i18n("NcbiExportDetailsView.files")}
+      </Typography.Title>
+      {bioSampleFiles.map((bioSampleFile: BioSampleFileDetails) => {
```

Comment:
The file display is good for now but we should remember to update it with the components I created in my sample details refactor stuff to keep it consistent

Reply:
Sounds good